### DISCUSSION TITLE
Show error message when an invalid field has been declared

### DISCUSSION THREAD

**Discussion by adrianthedev** [avo-hq/avo#443] [Feature requests]
When you declare a field that is invalid (`as: :string` instead of `as: :text`) you only get notified in the server console. We need to show that error better.

**Comment by Paul-Bob** [comment]
<img width="678" height="118" alt="image" src="https://github.com/user-attachments/assets/a9bb02b2-ba63-4231-a55e-0ae33e74f508" />
